src/de/privacy.md

```diff
@@ -28,21 +28,13 @@ Hierzu sowie zu weiteren Fragen zum Thema Datenschutz können Sie sich jederzeit
 
 ### Analyse-Tools und Tools von Dritt­anbietern
 
-Beim Besuch dieser Website kann Ihr Surf-Verhalten statistisch ausgewertet werden. Das geschieht vor allem mit sogenannten Analyseprogrammen.
-
-Detaillierte Informationen zu diesen Analyseprogrammen finden Sie in der folgenden Datenschutzerklärung.
+DFX erhebt keine zusätzlichen Daten um Surf-Verhalten statistisch auszuwerten und verwendet keine dieser sogenannten Analyseprogrammen.
 
 ## 2. Hosting
 
-### Hosting mit All-Inkl
-
-Wir hosten unsere Website bei All-Inkl. Anbieter ist die ALL-INKL.COM – Neue Medien Münnich, Inh. René Münnich, Hauptstraße 68, 02742 Friedersdorf (nachfolgend: All-Inkl). Details entnehmen Sie der Datenschutzerklärung von All-Inkl: [Datenschutzinformationen - ALL-INKL.COM](https://all-inkl.com/datenschutzinformationen/).
-
-Die Verwendung von All-Inkl erfolgt auf Grundlage von Art. 6 Abs. 1 lit. f DSGVO. Wir haben ein berechtigtes Interesse an einer möglichst zuverlässigen Darstellung unserer Website. Sofern eine entsprechende Einwilligung abgefragt wurde, erfolgt die Verarbeitung ausschließlich auf Grundlage von Art. 6 Abs. 1 lit. a DSGVO und § 25 Abs. 1 TTDSG, soweit die Einwilligung die Speicherung von Cookies oder den Zugriff auf Informationen im Endgerät des Nutzers (z. B. Device-Fingerprinting) im Sinne des TTDSG umfasst. Die Einwilligung ist jederzeit widerrufbar.
-
-#### Auftragsverarbeitung
+### Hosting mit GitHub
 
-Wir haben einen Vertrag über Auftragsverarbeitung (AVV) mit dem oben genannten Anbieter geschlossen. Hierbei handelt es sich um einen datenschutzrechtlich vorgeschriebenen Vertrag, der gewährleistet, dass dieser die personenbezogenen Daten unserer Websitebesucher nur nach unseren Weisungen und unter Einhaltung der DSGVO verarbeitet.
+Wir hosten unsere Website bei [GitHub](https://github.com/). Details entnehmen Sie der Datenschutzerklärung von GitHub: [[Datenschutzinformationen - ALL-INKL.COM](https://all-inkl.com/datenschutzinformationen/).](https://docs.github.com/de/site-policy/github-terms/github-terms-of-service) 
```

Comment:
Muss das so?
`[[Datenschutzinformationen - ALL-INKL.COM](https://all-inkl.com/datenschutzinformationen/).](https://docs.github.com/de/site-policy/github-terms/github-terms-of-service)`